integrated_channels/api/v1/canvas/views.py

```diff
@@ -0,0 +1,53 @@
+"""
+Viewsets for Canvas
+"""
+from rest_framework import permissions, status, viewsets
+from rest_framework.response import Response
+from edx_rbac.decorators import permission_required
+from enterprise.models import EnterpriseCustomerUser
+
+
+from integrated_channels.canvas.models import CanvasEnterpriseCustomerConfiguration
+
+from .serializers import CanvasEnterpriseCustomerConfigurationSerializer
+
+
+class CanvasConfigurationViewSet(viewsets.ModelViewSet):
+    serializer_class = CanvasEnterpriseCustomerConfigurationSerializer
```

Comment:
oh interesting you put this directly in the viewset instead of a mixin. I don't actually know what a mixin does, so seems fine?

Reply:
Yeah a mixin just effectively is a superclass so if you have fields on a mixin it'll apply to the  subclasses as well. Since we weren't re-using it outside of this viewset didn't make sense to externalize.